src/Umbraco.Tests/FrontEnd/UmbracoHelperTests.cs

```diff
@@ -78,5 +93,76 @@ public void Create_Encrypted_RouteString_From_Dictionary()
             
             Assert.AreEqual(expectedResult, result);
         }
+
+        [Test]
+        public void Truncate_By_Words_TagsAsContentOff()
+        {
+            var text = "Hello world, this is some text <a href='blah'>with a link</a>";
+
+            var helper = new UmbracoHelper();
+
+            var result = helper.TruncateByWords(text, 4, true, false).ToString();
+
+            Assert.AreEqual("Hello world, this is&hellip;", result);
+        }
+
+        [Test]
+        public void Truncate_By_Words_With_Tag_TagsAsContentOff()
+        {
+            var text = "Hello world, <b>this</b> is some text <a href='blah'>with a link</a>";
+
+            var helper = new UmbracoHelper();
+
+            var result = helper.TruncateByWords(text, 4, true, false).ToString();
+
+            Assert.AreEqual("Hello world, <b>this</b> is&hellip;", result);
```

Comment:
What is the difference between Truncate_By_Words_With_Tag_TagsAsContentOff and Truncate_By_Words_With_Tag_TagsAsContentOn ? They yield the exact same result so doesn't really seem like this boolean flag is doing anything especially with regards to these tests.  In fact, it kind of seems the same for all tests with ContentOn or ContentOff, the expectations are always the same so either these tests are not testing the correct behavior or the boolean flag doesn't actually do anything.